.g8/yesNoWithDetailsPage/app/views/$className$View.scala.html

```diff
@@ -0,0 +1,39 @@
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton,
+    govukDetails: GovukDetails
+)
+
+@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("$className;format="decap"$.title"))) {
+
+    @formHelper(action = routes.$className$Controller.onSubmit(mode), 'autoComplete -> "off") {
```

Comment:
Symbol literals are deprecated in Scala 2.13. Use Symbol("autoComplete") instead